### PR TITLE
Fixes #15888 - Remove attr_accessible calls

### PR DESCRIPTION
**Pull request by daviddavis**
Requires https://github.com/theforeman/foreman/pull/3659

### CODE REVIEW

File: app/models/service/containers.rb [hunk 1]
```diff
@@ -49,6 +50,19 @@ def create_container_object(wizard_state)
       container
     end
 
+    def container_attributes(wizard_state)
+      wizard_state.container_attributes.slice(:command, :repository_name,
```
Comment (daviddavis) [inline app/models/service/containers.rb]: Need to figure out how to use permit here since `wizard_state.container_attributes` is a Hash.

**Comment by dLobatog** [comment]
[test] let's see how this responds now that the core patch was merged

**Comment by daviddavis** [comment]
@dLobatog this should be ready for review now.

**Comment by dLobatog** [comment]
@daviddavis Looks good, not sure what changed since I left the last comment (when tests were green) but please take a look at the current failures, then I'll merge. Thanks!

**Comment by daviddavis** [comment]
@dLobatog Looks like the problem is that the UI registries controller we use `params[:docker_registry]` but in the api registries controller we're using `params[:registry]`. I've fixed it the param filter to account for this.

**Comment by dLobatog** [comment]
Merged as cc6d0d73e4893bcee000af9e2ba0258aaacdef36, thanks @daviddavis!

**Pull request closed without merging**